.github/pull_request_template.md

```
@@ -0,0 +1,16 @@
+<!-- Keep this light. Delete sections that don't apply. -->
+
+## What & why
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)